packages/manager/modules/pci/src/projects/project/storages/containers/container/addUser/translations/Messages_fr_FR.json

```diff
@@ -7,17 +7,17 @@
   "pci_projects_project_storages_containers_container_addUser_description_object_step_0": "Sélectionnez l'utilisateur à votre objet, S’il n’apparaît pas dans cette liste, veuillez le créer dans la section dédiée.",
   "pci_projects_project_storages_containers_container_addUser_description_container_step_1": "Définissez le rôle de votre utilisateur sur le conteneur {{value}}.",
   "pci_projects_project_storages_containers_container_addUser_description_object_step_1": "Définissez le rôle de votre utilisateur sur l'objet {{value}}.",
-
   "pci_projects_project_storages_containers_container_addUser_submit_label": "Confirmer",
   "pci_projects_project_storages_containers_container_addUser_next_label": "Suivant",
   "pci_projects_project_storages_containers_container_addUser_cancel_label": "Annuler",
   "pci_projects_project_storages_containers_container_addUser_back_label": "Retour",
-
   "pci_projects_project_storages_containers_container_addUser_right_admin": "Administrator",
   "pci_projects_project_storages_containers_container_addUser_right_readWrite": "Read and write",
   "pci_projects_project_storages_containers_container_addUser_right_readOnly": "Read only",
   "pci_projects_project_storages_containers_container_addUser_right_deny": "Deny",
-
+  "pci_projects_project_storages_containers_container_addUser_select_user_label": "Sélectionner un utilisateur",
+  "pci_projects_project_storages_containers_container_addUser_select_user_has_credential": "Credentials générés",
```

Comment:
Is "Credentials" also used in French ? Shouldn't it be "identifiants de connexion" or something like that ?

Reply:
Well see for me no, may be a delibrate chose, but i will check to be sure

Reply:
I got confirmation, it desired content.